easybuild/tools/build_log.py

```diff
@@ -105,12 +121,16 @@ def nosupport(self, msg, ver):
         """Print error message for no longer supported behaviour, and raise an EasyBuildError."""
         self.error("NO LONGER SUPPORTED since v%s: %s; see %s for more information" % (ver, msg, DEPRECATED_DOC_URL))
 
-    def error(self, msg, *args, **kwargs):
+    def standard_error(self, *args, **kwargs):
+        """Standard error method, just called error method of parent."""
+        super(EasyBuildLog, self).error(*args, **kwargs)
+
+    def easybuild_error(self, msg, *args, **kwargs):
         """Print error message and raise an EasyBuildError."""
-        newMsg = "EasyBuild crashed with an error %s: %s" % (self.caller_info(), msg)
-        fancylogger.FancyLogger.error(self, newMsg, *args, **kwargs)
+        new_msg = "EasyBuild crashed with an error %s: %s" % (self.caller_info(), msg)
```

Comment:
Note that this still has the same problem with varargs that Jens reported.  You should interpolate `args` into `msg`, e.g.::

    
+ new_msg = "EasyBuild crashed with an error %s: %s" % (self.caller_info(), (msg % args))